arrow/formatter.py

```diff
@@ -122,18 +121,22 @@ def _format_token(self, dt, token):
 
         if token in ["ZZ", "Z"]:
             separator = ":" if token == "ZZ" else ""
-            tz = dateutil_tz.tzutc() if dt.tzinfo is None else dt.tzinfo
-            total_minutes = int(util.total_seconds(tz.utcoffset(dt)) / 60)
+            tz: Union[dateutil_tz.tzutc, tzinfo] = (
+                dateutil_tz.tzutc() if dt.tzinfo is None else dt.tzinfo
+            )
+            total_minutes = int(util.total_seconds(tz.utcoffset(dt)) / 60)  # type: ignore
 
             sign = "+" if total_minutes >= 0 else "-"
             total_minutes = abs(total_minutes)
             hour, minute = divmod(total_minutes, 60)
 
-            return "{}{:02d}{}{:02d}".format(sign, hour, separator, minute)
+            return f"{sign}{hour:02d}{separator}{minute:02d}"
 
         if token in ("a", "A"):
             return self.locale.meridian(dt.hour, token)
 
         if token == "W":
             year, week, day = dt.isocalendar()
-            return "{}-W{:02d}-{}".format(year, week, day)
+            return f"{year}-W{week:02d}-{day}"
+
+        return None  # To appease the pymy
```

Comment:
Rather than returning None to appease Mypy, should we just raise an exception and give a helpful message to users? Like an `ArrowFormatterError` or something. Currently, if you pass in a bad token, you get garbage:

```python
>>> arrow.now().format("derps")
'7erp26'
```